src/Auth0.php

```diff
@@ -392,6 +423,13 @@ public function getLoginUrl(array $params = [])
             $this->stateHandler->store($auth_params['state']);
         }
 
+        $auth_params['nonce'] = self::getNonce();
```

Comment:
Do not allow developers to pass in a nonce, always set one.

Reply:
Developers could specify their own nonce if they want so. Just like they can specify a custom state as well (line above this one). If the nonce value is not present by then, that's when you generate a random one as default.

Reply:
Why would a developer need to specify a nonce? State makes sense, they want that value on the return, but why nonce?